pack/tcu/bbt_encounter.json

```diff
@@ -0,0 +1,404 @@
+		"code": "05327",
+		"doom": 6,
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "...the boundless daemon-sultan Azathoth, whose name no lips dare speak aloud, and who gnaws hungrily in inconceivable, unlighted chambers beyond time amidst the muffled, maddening beating of vile drums and the thin, monotonous whine of accursed flutes...\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"illustrator": "Lenka Šimečková",
+		"name": "It Awaits",
+		"pack_code": "bbt",
+		"position": 327,
+		"quantity": 1,
+		"stage": 2,
+		"text": "Each location is connected to each location adjacent to it.\nDo not remove doom from enemies when this agenda advances.",
+		"type_code": "agenda"
+	},
+	{
+		"back_name": "Azathoh Stirs",
+		"back_text": "Each [[Cultist]] enemy commits ritual suicide. Discard each [[Cultist]] enemy in play and move all doom from them to Azathoth.\nThe lead investigator must search the encounter deck and discard pile for a copy of Daemonic Piping and draw it.\nThere is no longer an agenda deck, but the scenario is not over. Until the end of the scenario, doom that would be placed on the current agenda is instead placed on Azathoth, and all doom in play is considered to be on Azathoth.\n<hr><blockquote><i>You come across another circle of Azathoth's worshippers, led by a hooded, shapeless form which holds an ancient lantern high above its head. The lantern's dim grey light scarcely pierces the realm's darkness. Somehow, the phantom is familiar to you; some vague sense of kinship drives you onward.</i></blockquote>\nCheck Campaign Log. If <i>Corn Husk Doll</i> is listed under \"Mementos Discovered,\" the creature recognizes you, as well. It peers at you curiously, and the circle of prayer is silenced; remove 1 doom from Azathoth.\nOtherwise, the creature barely notices you, and the circle resumes its chanting.",
+		"code": "05328",
+		"doom": 8,
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "...that leering, tittering shantak coursed on impetuous and relentless, flapping its great slippery wings in malignant joy, and headed for those unhallowed pits whither no dreams reach... \n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"illustrator": "Lenka Šimečková",
+		"name": "The Final Countdown",
+		"pack_code": "bbt",
+		"position": 328,
+		"quantity": 1,
+		"stage": 3,
+		"text": "Each location is connected to each location adjacent to it.\nDo not remove doom from enemies when this agenda advances.",
```

Comment:
Please add in italics: '(Beware - the scenario will not end when this agenda advances, but the doom of the world will be close at hand.)'

Reply:
... at the end of the line, I mean!